addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -516,6 +540,14 @@ registry.slider = publicWidget.Widget.extend({
     _onContentChanged: function (ev) {
         this._computeHeights();
     },
+    /**
+     * Redirects a carousel control click on the active slide.
+     *
+     * @private
+     */
+    _onControlClick() {
+        this.$target[0].querySelector(".carousel-item.active").click();
+    },
```

Comment:
Do you actually need to do that even though you remove the data-slide and data-slide-to attributes? :open_mouth:

Reply:
Yes because the options do not start if we click on  the "controls" element, it only does when clicking on the slide :slightly_smiling_face: So the click listener that is in the options will never be called (since it was not even added in the first place) 
So if we click on them without redirecting the click on the active slide, it will just do nothing (since the carousel behavior was removed) making them look broken

I agree it is a bit "hackish", but I didn't find another way to start the options :shrug:

Reply:
> Yes because the options do not start if we click on the "controls" element, it only does when clicking on the slide 🙂 

Yes but I would have imagined that the reason was bootstrap preventing event propagation when there is the `data-slide` or `data-slide-to` attribute. If not that, what is the reason for the slide to not be considered as "clicked" when you click on those controls?

Reply:
To write here what I explained IRL: 
The reason the option does not start is because of the `o_we_no_overlay` class that is added on the carousel controls -> this class prevents `activate_snippet` to be called when clicking (see `_notActivableElementsSelector` in `snippets.editor.js`) :slightly_smiling_face: 

When removing this class, clicking on the controls indeed starts the option. However, as the click event to slide the carousel is therefore "lost" (since it was used to activate the snippet), it does not slide the carousel => we need to click once again in order for the listener in the options to slide. 
As a solution, we could reactivate the carousel controls, meaning that they will slide the carousel but once again the listener will not be called the first time, so this sliding will not be encapsulated in the mutex (and I would still need to deactivate the controls after that anyway or I wouldn't be able to encapsulate in the mutex as Bootstrap handles the click on its side)

For the reason why it needs to be mutexed, it's because if it is not, other operations could happen during a sliding, so if we undo this operation, it could break the carousel if the undo ends up in the middle of the sliding for example, leaving it with inconsistent classes. This is mainly hypothetical as we need to do that really quickly but since the point of the PR was to avoid race condition, I think it is needed :slightly_smiling_face: 
Also, if not mutexed, the undo of the first sliding is not done correctly.